impl/src/test/java/io/kroxylicious/testing/kafka/KafkaClusterTest.java

```diff
@@ -69,6 +71,49 @@ public void kafkaClusterZookeeperMode() throws Exception {
         }
     }
 
+    @Test
+    public void kafkaClusterZookeeperAddBroker() throws Exception {
+        int brokersNum = 1;
+        try (var cluster = KafkaClusterFactory.create(KafkaClusterConfig.builder()
+                .testInfo(testInfo)
+                .brokersNum(brokersNum)
+                .kraftMode(true)
```

Comment:
There is a mismatch between the name of the test using `Zookeeper` and the configuration of the cluster `kraftMode(true)` that should be `kraftMode(false)`